templates/dashboard/index.tpl

```diff
@@ -14,6 +14,12 @@
 		{translate key="navigation.submissions"}
 	</h1>
 
+	{if $currentContext->getData('disableSubmissions')}
+		<notification>
+			{translate key="manager.setup.disableSubmissions.notAccepting"}
+		</notification>
+	{/if}
+
```

Comment:
I suggest getting rid of the notification here. On reflection, I think it's not necessary to see this every time the submissions list is visited. Let's reserve it for the settings pages.

If you prefer to keep it in the submissions page, let's make sure that it is only shown to journal managers. You can use `in_array(ROLE_ID_MANAGER, $userRoles)` in the template to check if someone is a journal manager.